### ISSUE TITLE
[Bug] Incompatible Tweaks Detected Pop-up keeps showing

### ISSUE THREAD

**Issue by ExtNinja2099** [YTLitePlus/YTLitePlus#395]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### iOS/iPadOS version

iOS 17.6.2

### Device model

iPhone 15 Plus

### Sideload method

Sideloadly

### Tweak version

YTLitePlus - v19.34.2-5.0.1

### Do you use YouTube Premium?

❌ No, I don't use YouTube Premium

### Are you using the newest version of YTLitePlus?

✅ Yes, I'm using the latest version

### Does the issue happen with the official YouTube from AppStore?

❌ No, I can't reproduce this issue in the official YouTube from AppStore

### Method used to build the iPA

buildapp.yml (GitHub Actions)

### Describe the bug

![image](https://github.com/user-attachments/assets/668eb241-8306-4d1f-8787-6d0a8cadd2c9)
This pop up keeps appearing after opening/closing the app even if I pressed "I Accept All Risks"
Like this pop is persistent even if I resetted and reinstalled the app to get default settings in YTLitePlus
This wasn't appearing for 4.0.1 and I don't know how to remove this pop up
Which setting is causing this issue?

### Steps to reproduce the issue

1. Uninstall YTLitePlus
2. Install latest YTLitePlus v19.34.2-5.0.1
3. After installation, open the app
4. The pop up appears and press "I Accept All Risks"
5. Close the open and reopen the app
6. The pop should appear again

**Comment by bhackel** [comment]
Yes, there is no setting to disable it from what I can tell. I am working on removing it on the v5 branch, but it is not perfect yet.

**Comment by ExtNinja2099** [comment]
Even after disabling and trying to remove iSponsorblock in settings, the popup still appears

**Comment by dayanch96** [comment]
> Yes, there is no setting to disable it from what I can tell. I am working on removing it on the v5 branch, but it is not perfect yet.

Please respect my choices. If you try to remove it, I'll make sure that YouTube Plus won't work with your tweak at all.

**Comment by bhackel** [comment]
I removed iSponsorBlock and the overlay stayed. I’m thinking that there’s another tweak causing it.

**Comment by Balackburn** [comment]
> I removed iSponsorBlock and the overlay stayed. I’m thinking that there’s another tweak causing it.

I think any tweak injected with YTPlus cause this popup

**Comment by bhackel** [comment]
I see

**Comment by bhackel** [comment]
Well I’m moving on with my life then. Thanks for the fun times everyone

**Comment by Balackburn** [comment]
> Well I’m moving on with my life then. Thanks for the fun times everyone

Why just because their is popup for incompatible tweaks? Can't you just hide it with an option?

**Comment by Balackburn** [comment]
@bhackel @arichornloverALT Are you both giving up on YTLitePlus completely?

**Comment by bhackel** [comment]
It just feels like a waste of time for me to develop something that stops another developer's code from working as intended. I would much prefer to work on something that provides value to people. I'm busy with some life stuff right now, I'll let you know what I think in a few days.

**Comment by CiroHoodLove** [comment]
@bhackel @arichornloverALT 
Damn i really hope y’all don’t split and find a solution to work in harmony , because i can tell you that your work provides a great help for people like myself , i switched to iPhone a couple months ago which was very limited in terms of sideloading compared to the android , your (**including all devs repectfully**) tweak was a huge help and very well made one which offered all features from which some i didn’t know where possible to achieve, so i gotta show my appreciation even if y’all decide to not continue with this project , i wish y’all the best

**Comment by KieranBLK** [comment]
Hmmm, odd -- I installed 5.0.1 and launched it no problem. Didn't get any incompatibility pop up or anything.

**Comment by ExtNinja2099** [comment]
> Hmmm, odd -- I installed 5.0.1 and launched it no problem. Didn't get any incompatibility pop up or anything.

Well bhackel has pushed new commits to remove it 3 hours ago lol
So the new build u have won't have it. But it displays a toast pop up showing (Bypassing pop-up)
It's technically gone now and have that toast pop up instead
At least I don't have to keep pressing "I Accept All Risks" everytime I open the app
Thanks [bhackel](https://github.com/bhackel)

**Comment by garshmo** [comment]
Ambient mode not working at all and no option for it in video player on newest release

**Comment by ExtNinja2099** [comment]
> Ambient mode not working at all and no option for it in video player on newest release

Copying and selecting text I replies don't work either. But we'll see

**Comment by mustafacakirogludev** [comment]
> But it displays a toast pop up showing (Bypassing pop-up)

I think this is quite risky. The original developer (dayanch96) said if they remove it, he'll break the tweaks completely in this very thread.  bhackel and arichornlover just add additional tweaks on top of dayanch's repo, he is the one providing the main development and bugfixes. Hopefully he does not stop developing YTLite or we are in a huge trouble, we can live without additional tweaks, I'd hate to lose everything.

**Comment by castdrian** [comment]
Time to fork and remove the stupid overlay to piss off @dayanch96 the same way they've pissed off everyone in this thread.

**Comment by dayanch96** [comment]
> Time to fork and remove the stupid overlay to piss off @dayanch96

Well, good luck with that. Even better, why not gather all the haters into one team, create your own YouTube tweak, and just forget YTPlus even exists?

It’s hilarious how deeply you all despise me, yet somehow can’t function without my tweak. Make it make sense

**Comment by castdrian** [comment]
most people me included use uyouenhanced/plus so you should calm the delusions of your non existing importance, just an excellent opportunity to infill a prick with some spite

**Comment by Balackburn** [comment]
> most people me included use uyouenhanced/plus so you should calm the delusions of your non existing importance, just an excellent opportunity to infill a prick with some spite

Why do you come here to insult hard working developers who offer their work for free? 

Go away you miserable loser.

**Comment by dayanch96** [comment]
> most people me included use uyouenhanced/plus so you should calm the delusions of your non existing importance, just an excellent opportunity to infill a prick with some spite

Good for you. Using another tweak as 'a defense' is fine, but hanging around a repo directly tied to my work speaks for itself. By resorting to personal attacks and trying to provoke others, you're only showing a need for attention. Maybe it's worth considering a talk with a therapist to figure out what’s really going on. Consider it as friendly advice.

**Comment by castdrian** [comment]
My therapist already knows plenty not that much left to reveal but going the ooh aaahh mental health route just proves that your buttons can be pushed way to easily, as for my hanging around I was debugging if a tweak I wrote was gonna accidentally explode anything when combined with as many others as possible, stumbled across your hideous behavior on accident if you will

**Comment by IISAKNIFE** [comment]
> > most people me included use uyouenhanced/plus so you should calm the delusions of your non existing importance, just an excellent opportunity to infill a prick with some spite
> 
> Good for you. Using another tweak as 'a defense' is fine, but hanging around a repo directly tied to my work speaks for itself. By resorting to personal attacks and trying to provoke others, you're only showing a need for attention. Maybe it's worth considering a talk with a therapist to figure out what’s really going on. Consider it as friendly advice.

Dude why do you make it pop up EVERY TIME. I accept the risks, accept that I dont want you bugging me EVERY SINGLE TIME

**Comment by KieranBLK** [comment]
> Dude why do you make it pop up EVERY TIME. I accept the risks, accept that I dont want you bugging me EVERY SINGLE TIME

Very likely because he doesn't want people just glossing over it, or it's an intentional dissuasion to get his "haters" from using the tweak at all. Which so far has worked more in his favor so... 🤷🏽‍♂️

**Comment by kartavkun** [comment]
Any progress?

**Comment by dayanch96** [comment]
> @dayanch96 got caught multiple times over the year distributing malicious codes (otp Steelers , crypto Steelers & clipboard crypto address changers ) and Tweeks in various repos. so i won't be comfortable trusting any of his Tweeks or anything made based on him.

That’s why I always say to use my tweaks only from my repos. I’m not responsible of that kind of scam that implemented by third parties. I also got these reports, especially from InfusePlus users.

If you don’t believe me (and you don’t have to believe me), you’re more than welcome try yourself or ask someone who can disassemble my tweaks and check personally of such scams

**Comment by dayanch96** [comment]
> Any progress?

coming with 5.1 soon
